stackslib/src/chainstate/stacks/db/blocks.rs

```diff
@@ -2152,7 +2154,7 @@ impl StacksChainState {
         block_hash: &BlockHeaderHash,
     ) -> Result<Vec<ConsensusHash>, Error> {
         let qry = "SELECT consensus_hash FROM staging_blocks WHERE anchored_block_hash = ?1";
-        let args: &[&dyn ToSql] = &[block_hash];
+        let args: &[&dyn ToSql] = params![block_hash];
```

Comment:
Since you are using `params!()` everywhere now, does it still make sense to keep the explicit type declarations? Why not do this

```suggestion
        let args = params![block_hash];
```

Reply:
I've seen there are places where it was used and places were it wasn't. I went with having it explicit. I can remove the explicit type as the `params!` macro should always return that type https://docs.rs/rusqlite/latest/rusqlite/macro.params.html.

Reply:
done https://github.com/stacks-network/stacks-core/pull/4948/commits/97bace46a5f20c46fce42f139fe4f79cb973b32c